src/Forms/AzureMigrateExportMainForm.cs

```diff
@@ -890,6 +893,11 @@ public bool IsAvsBusinessProposalSelected()
             return (ConfigurationFormObj.GetBusinessProposal() == BusinessProposal.AVS.ToString());
         }
 
+        public bool IsImportAndComprehensiveProposalSelected()
+        {
+            return ((ConfigurationFormObj.GetBusinessProposal() == BusinessProposal.Comprehensive.ToString()) && (ConfigurationFormObj.IsImportRadioButtonChecked()));

```

Comment:
length of a line should not exceed 100 columns

return (
(ConfigurationFormObj.GetBusinessProposal() == BusinessProposal.Comprehensive.ToString()) &&
(ConfigurationFormObj.IsImportRadioButtonChecked()));